src/main/webapp/test/mashup.jsp

```diff
@@ -14,8 +14,8 @@
     <title>Webpage Compilation - TEAMMATES</title>
 
     <link rel="shortcut icon" href="/favicon.png">
-    <link type="text/css" href="<%= FrontEndLibrary.BOOTSTRAP_CSS %>">
-    <link type="text/css" href="<%= FrontEndLibrary.BOOTSTRAP_THEME_CSS %>">
+    <link type="text/css" href="<%= FrontEndLibrary.BOOTSTRAP_CSS %>" rel="stylesheet">
+    <link type="text/css" href="<%= FrontEndLibrary.BOOTSTRAP_THEME_CSS %>" rel="stylesheet">
```

Comment:
I'm curious what is the purpose of making the `MashupPageUiTest` workable again when it isn't maintained? In its current state, its missing a few pages as well isn't it?

Reply:
And if you let it be broken forever, you're essentially declaring it dead. Prematurely, if I may add.
Whether it is still used/maintained is out of the question, but at least I'm giving the chance for this page to be usable again if the need arises.